activerecord/test/models/season.rb

```diff
@@ -0,0 +1,3 @@
+class Season < ActiveRecord::Base
+  self.primary_key = nil
+end
```

Comment:
I looked for in all test models, in the schema.rb and I didn't find none model without primary key.